### PR TITLE
Improving content on Canada.ca

### PR DESCRIPTION
**Pull request by prycrane**
**Trello**

- https://trello.com/c/f3reEdEW

**English PR**

- https://github.com/canada-ca/design-system/pull/394

**Branch**

- https://github.com/canada-ca/systeme-conception/tree/improving-content-fr

**Preview**

- https://deploy-preview-256--systeme-conception-canada-ca.netlify.app/partenaires/
- https://deploy-preview-256--systeme-conception-canada-ca.netlify.app//a-propos/

**Design document**

- https://docs.google.com/document/d/1nKJ8hC0x5-l_7tgutGOM7KjZ1jDgsxOztuA7JIEWGew/edit#heading=h.c27giw6xp4p

### CODE REVIEW

**Comment by donohuech** [comment]
@prycrane on the English page: https://deploy-preview-394--design-system-canada-ca.netlify.app/partners/ we have the dates for each project but on the French page we don't https://deploy-preview-256--systeme-conception-canada-ca.netlify.app/partenaires/

**Review by donohuech — CHANGES_REQUESTED** [review]

File: partenaires/project-09.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+altLangPage: "https://design.canada.ca/partners/project-09.html"
+date: 2024-01-02
+dateModified: 2024-01-02
+description: "Projet partenaire : Make it easier for Canadian parents to understand and apply for maternity and parental benefits."
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-09.md:5]: Done

File: partenaires/project-09.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  </div>
+</div>
+<h2>Scénarios de tâches testés et améliorés</h2>
+<ul class="lst-spcd">
+  <li>Anne attend un bébé dans six semaines. Elle veut travailler jusqu'à sa date d'accouchement et prendre un congé après. Elle présentera une demande au gouvernement en vue de recevoir de l'argent pendant son congé. Quand doit-elle envoyer sa demande pour que tout soit prêt avant la naissance du bébé?</li>
+  <li>Le médecin de Maya lui recommande de prendre congé cinq semaines avant la naissance de son bébé. Elle n'a pas de journée de congé ni de congés de maladie restants à son travail. Doit-elle utiliser son congé de maternité ou a-t-elle d'autres options?</li>
+  <li>Vous avez travaillé à temps plein au cours de la dernière année et attendez un bébé. Comment pouvez-vous savoir si votre rémunération est considérée comme « assurable » de sorte que vous puissiez demander un congé parental pour vous occuper du bébé?</li>
+  <li>iane attend un bébé. Elle veut prendre un congé aussi long que possible. Son partenaire veut également prendre quatre semaines de congé immédiatement après la naissance. Quelle est la durée maximale (du début à la fin) du congé que Diane peut prendre?</li>
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-09.md:48]: fixed

File: partenaires/project-10.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+altLangPage: "https://design.canada.ca/partners/project-10.html"
+date: 2024-01-03
+dateModified: 2024-01-03
+description: "Projet partenaire : Combine information from across departments so that Canadians easily get the whole picture on what the Government of Canada requires when starting a business."
+title: "Activités de démarrage d'entreprise"
+---
+<p>Combiner les renseignements provenant de l'ensemble des ministères afin que les Canadiens puissent obtenir facilement le portrait d'ensemble des éléments dont le gouvernement du Canada a besoin au moment de démarrer une entreprise.</p>
+<h2>Ministère partenaire</h2>
+<ul>
+  <li>Services publics et Approvisionnement Canada (SPAC)</li>
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-10.md:11]: Done

File: partenaires/project-10.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+</div>
+<h2>Scénarios de tâches testés et améliorés</h2>
+<ul class="lst-spcd">
+  <li>Imaginez que vous allez démarrer une entreprise à domicile à Winnipeg, à temps partiel, à titre de consultant. Renseignez-vous sur les étapes juridiques essentielles de ce processus.
+Vous voulez nommer votre entreprise &laquo;&nbsp;SOMOS &nbsp;&raquo;. Vérifiez s'il est légal ou illégal pour vous de commencer à utiliser ce nom.
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-10.md:46]: done.

File: partenaires/project-10.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+Vous pensez gagner environ 1 000 $ par mois de votre nouvelle entreprise pendant les six premiers mois. Vérifiez si vous devez facturer la TPS à vos clients.
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-10.md:47]: Done.

File: partenaires/project-10.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+Votre premier employé commence à travailler le mois prochain. Vérifiez si vous devez appliquer une mesure officielle avant ou peu après avoir émis son premier chèque de paye.
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-10.md:48]: Done.

File: partenaires/project-13.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
+---
+altLangPage: "https://design.canada.ca/partners/project-13.html"
+date: 2024-01-03
+dateModified: 2024-01-03
+description: "Projet partenaire : Aider les personnes à se préparer à l’examen pour la citoyenneté canadienne."
+title: "Se préparer à passer l'examen de citoyenneté"
+---
+<p>Make it easier for applicants to study for the Canadian citizenship test.</p>
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-13.md:8]: Fixed

File: partenaires/project-13.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+</div>
+<h2>Scénarios de tâches testés et améliorés</h2>
+<ul class="lst-spcd">
+  <li>Vous avez présenté votre demande de citoyenneté. Que devrez-vous faire d'autre pour devenir citoyen canadien?
+    Quelles sont les différentes façons d'étudier pour l'examen pour la citoyenneté canadienne?
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-13.md:47]: Fixed

File: partenaires/project-13.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+    Y aura-t-il des questions pour évaluer vos compétences linguistiques en anglais ou en français dans le cadre de l'examen pour la citoyenneté?
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-13.md:48]: Fixed.

File: partenaires/project-13.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+    Selon le guide de l'étude sur la citoyenneté, en quelle année le drapeau canadien actuel est-il devenu le drapeau officiel du Canada?
+    Selon le guide de l'étude sur la citoyenneté, que signifie le mot Inuit?
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-13.md:50]: Fixed

File: partenaires/project-13.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+    Que devez-vous emporter avec vous lorsque vous allez passer l'examen de citoyenneté?</li>
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-13.md:51]: Fixed

File: partenaires/project-14.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+---
+<p>Veiller à ce que les Canadiens puissent ouvrir et gérer plus facilement leurs comptes d'entreprise auprès de l'ARC.</p>
+<h2>Ministère partenaire</h2>
+<ul>
+  <li>Agence du revenu du Canada (ARC)</li>
+</ul>
+<h2>Pourquoi ce travail est important</h2>
+<ul>
+  <li>Améliorer la convivialité des options de libre-service réduit la charge de travail des entreprises en simplifiant la gestion des renseignements sur les comptes.</li>
+  <li>ARediriger les transactions simples vers des options numériques moins coûteuses réduit les coûts de soutien pour l'ARC.</li>
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-14.md:16]: Fixed

File: partenaires/project-19.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
+---
+altLangPage: "https://design.canada.ca/partners/project-19.html"
+date: 2017-01-03
+dateModified: 2017-01-03
+description: "Projet partenaire : Aider les Canadiens et Canadiennes à comprendre les avis relatifs aux conditions météorologiques dangereuses. Date : Juillet 2017"
+title: "Renseignements météorologiques"
+---
+<p>Make it easier for Canadians to understand notifications about dangerous weather conditions.</p>
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-19.md:8]: Fixed

File: partenaires/project-09.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
+---
+altLangPage: "https://design.canada.ca/partners/project-09.html"
+date: 2024-01-02
+dateModified: 2024-01-02
+description: "Projet partenaire : Make it easier for Canadian parents to understand and apply for maternity and parental benefits."
+title: "Rendre les prestations de maternité et les prestations parentales de l’assurance-emploi plus faciles à trouver et à comprendre pour les Canadiens et Canadiennes qui prévoient agrandir leur famille. Date : 2019"
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-09.md]: fixed

File: partenaires/project-10.md [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+Votre entreprise doit commencer à percevoir la TPS. Vérifier quelle est la première mesure officielle que vous devez appliquer.
```
[the remark this answers is not in the document]
Comment (prycrane) [reply partenaires/project-10.md]: Fixed

**Review by donohuech — APPROVED** [review]

**Pull request merged**